src/docs/PCA/references/Storm.adoc

```diff
@@ -0,0 +1,29 @@
+This service allows to deploy through ProActive Cloud Automation (PCA) Portal a storm platform.
+
+This deployment is done in one step 
+
+- *Deploy Storm* 
+
+image::PCA_storm.png[align=center]
+
+At the end, the instance state is at RUNNING. 
+One action is possible at this step; it is to finish the instance. 
+
+image::PCA_storm1.png[align=center]
+
+You can also access to the instance endpoint (Storm UI).
+
+image::PCA_storm2.png[align=center]
+
+
+- *Finish Storm* 
+
+The running instance can be finished by executing the available action when the instance is at the RUNNING state.
+You have to give the instance name you want to finish.
```

Comment:
I think this should be check with Nebil this is probably an error during the wf conception.